dask/array/tests/test_random.py

```diff
@@ -50,6 +50,15 @@ def test_determinisim_through_dask_values():
            [v for k, v in sorted(samples_2.dask.items())]
 
 
+def test_randomstate_consistent_names():
+    state1 = da.random.RandomState(42)
+    state2 = da.random.RandomState(42)
+    assert sorted(state1.normal(size=(100, 100), chunks=(10, 10)).dask) ==\
+           sorted(state2.normal(size=(100, 100), chunks=(10, 10)).dask)
+    assert sorted(state1.normal(size=100, loc=4.5, scale=5.0, chunks=10).dask) ==\
+           sorted(state2.normal(size=100, loc=4.5, scale=5.0, chunks=10).dask)
```

Comment:
It would be good also to have a negative test.  For example the following should be the case

```python
assert sorted(state1.normal(size=100, loc=4.5, scale=5.0, chunks=10).dask) !=\
        sorted(state!.normal(size=100, loc=4.5, scale=5.0, chunks=10).dask)
```